script/brave_license_helper.py

```diff
@@ -29,6 +29,10 @@ def AddBraveCredits(prune_paths, special_cases, prune_dirs, additional_paths):
         os.path.join('brave', 'third_party', 'android_deps'),
         os.path.join('brave', 'third_party', 'blink'),
         os.path.join('brave', 'third_party', 'libaddressinput'),
+
+        # misc tools
+        os.path.join('brave', 'vendor', 'depot_tools'),
+        os.path.join('brave', 'vendor', 'gn-project-generators')
```

Comment:
This one is actually under a [different license](https://github.com/brave/gn-project-generators/blob/master/LICENSE) so we should include it in the [`special_cases` data structure](https://github.com/brave/brave-core/blob/87be00e446f49669936a447f3c289d295733ad90/script/brave_license_helper.py#L36):
```
        os.path.join('brave', 'vendor', 'gn-project-generators'): {
            "Name": "Project generators for GN",
            "URL": "https://github.com/brave/gn-project-generators",
            "License": "BSD-3-Clause",
            "License File": "/brave/vendor/gn-project-generators/LICENSE",
      }
```

Reply:
do we need to include it? It isn't used in Brave, it's just a tool that gets used to generate an xcode workspace from the gn files

Reply:
Resolving in favor of https://github.com/brave/brave-core/pull/6278#discussion_r463879545.